tests/TaskDynamicsTest.cpp

```diff
@@ -78,10 +82,60 @@ TEST_CASE("Test Proportional")
   f->updateValue();
   tdi->updateValue();
 
+  FAST_CHECK_EQ(td.order(), task_dynamics::Order::One);
   FAST_CHECK_EQ(tdi->order(), task_dynamics::Order::One);
   FAST_CHECK_EQ(tdi->value()[0], -kp*(36 - rhs[0]));  // -kp*||(1,2,3) - (1,0,-3)||^2 - 2^2 - rhs
   FAST_CHECK_UNARY(tdi->checkType<task_dynamics::P>());
   FAST_CHECK_UNARY_FALSE(tdi->checkType<task_dynamics::Constant>());
+
+  kp = 3;
+  dynamic_cast<task_dynamics::P::Impl*>(tdi.get())->gain(kp);
+  tdi->updateValue();
+  FAST_CHECK_EQ(tdi->value()[0], -kp * (36 - rhs[0]));  // -kp*||(1,2,3) - (1,0,-3)||^2 - 2^2 - rhs
+}
+
+TEST_CASE("Test Proportionnal gain types")
```

Comment:
Proportionnal -> Proportional (a few times in the file)

Reply:
Fixed in 420194e